student.rb

```diff
@@ -0,0 +1,15 @@
+require './person'
+
+class Student < Person
+  def initialize(classroom)
+    super(name, age, parent_permission)
+    @classroom = classroom
+  end
```

Comment:
In order to call `super` here without errors, kindly include age, name and parent_permission as the constructor parameters and include the default parameters too.